### PR TITLE
error on gauge reporter

### PR DESCRIPTION
**Pull request by twillouer**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -105,6 +105,17 @@
...
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<version>2.7.2</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
+			<version>3.6.1</version>
```
Comment (dw-alex) [inline pom.xml:117]: scope test

File: src/test/java/io/github/hengyunabc/metrics/test/ZabbixReporterTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            @Override
+            public Long getValue()
+            {
+                return 1L;
+            }
+        });
+        // When
+        instance.report();
+
+        // When
```
Comment (a14n) [inline src/test/java/io/github/hengyunabc/metrics/test/ZabbixReporterTest.java]: then ?

**Comment by twillouer** [comment]
Hello,

can you take a look ?

**Comment by hengyunabc** [comment]
@twillouer Sorry, I will process this at the weekend.

**Pull request merged**